### PR TITLE
osd/OSD: make 'ceph tell osd.x get_command_descriptions' work.

### PR DESCRIPTION
**Pull request by majianpeng**
Signed-off-by: Jianpeng Ma <jianpeng.ma@intel.com>

### CODE REVIEW

**Comment by liewegas** [comment]
this breaks ceph bench.

./ceph tell osd.0 bench 1 2097152

<pre>
2016-05-05 11:51:43.110509 7fe0e641a700  1 -- 127.0.0.1:6804/21229 <== client.4101 127.0.0.1:0/3938302392 2 ==== command(tid 2: {"count": 1, "prefix": "bench", "size": 2097152}) v1 ==== 72+0+0 (3573759197 0 0) 0x7fe01c000910 con 0x7fe028001d00
2016-05-05 11:51:43.110550 7fe0e641a700 10 osd.0 5 do_waiters -- start
2016-05-05 11:51:43.110556 7fe0e641a700 10 osd.0 5 do_waiters -- finish
2016-05-05 11:51:43.110558 7fe0e641a700 20 osd.0 5 _dispatch 0x7fe01c000910 command(tid 2: {"count": 1, "prefix": "bench", "size": 2097152}) v1
2016-05-05 11:51:43.110575 7fe0e641a700 10 osd.0 5 do_waiters -- start
2016-05-05 11:51:43.110577 7fe0e641a700 10 osd.0 5 do_waiters -- finish
2016-05-05 11:51:43.110642 7fe05ffff700 20 osd.0 5 do_command tid 2 [{"count": 1, "prefix": "bench", "size": 2097152}]
2016-05-05 11:51:43.110711 7fe05ffff700  1 osd.0 5  bench count 1 bsize 0 bytes
...
</pre>
and it runs forever

**Comment by tchaikov** [comment]
tested at http://pulpito.ceph.com/kchai-2016-06-12_09:44:10-rados-wip-kefu-testing---basic-mira/

http://pulpito.ceph.com/kchai-2016-06-12_09:44:10-rados-wip-kefu-testing---basic-mira/254583/ addressed by https://github.com/ceph/ceph-qa-suite/pull/1052

File: src/test/osd/osd-bench.sh [hunk 1]
```diff
@@ -77,6 +77,16 @@ function TEST_bench() {
     # default values should work
     #
     ceph tell osd.0 bench || return 1
+
+    #
+    #test object_size < block_size
```
Comment (tchaikov) [inline src/test/osd/osd-bench.sh]: could you add a space after `#`?

**Comment by majianpeng** [comment]
@tchaikov : update

**Comment by tchaikov** [comment]
tested at 

- http://pulpito.ceph.com/kchai-2016-06-15_04:31:07-rados-wip-kefu-testing---basic-mira/
- http://pulpito.ceph.com/kchai-2016-06-15_10:14:46-rados-wip-kefu-testing---basic-mira/
- http://pulpito.ceph.com/kchai-2016-06-17_02:37:57-rados-wip-kefu-testing---basic-mira/

**Comment by tchaikov** [comment]
lgtm.

**Pull request merged**